conanfile.py

```diff
@@ -7,13 +7,14 @@ class SiloRecipe(ConanFile):
 
     requires = [
         "boost/1.82.0",
+        "duckdb/0.8.1",
         "poco/1.12.4",
+        "hwloc/2.9.3",
```

Comment:
On my machine, I get an version conflict when running with ./build_with_conan.py
`ERROR: Version conflict: onetbb/2021.9.0->hwloc/2.9.2, ->hwloc/2.9.3.`
When I use 2.9.2 then it works. Do you have the same problem?

Reply:
I will check again

Reply:
Is it only on your machine or on the server?